Apps/W1/EDocumentsConnector/app/src/Continia/Metadata/NetworkProfile.Table.al

```diff
@@ -0,0 +1,73 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+namespace Microsoft.EServices.EDocumentConnector.Continia;
+
+table 6394 "Network Profile"
+{
+    Caption = 'Network Profile';
+    DataClassification = SystemMetadata;
+    DrillDownPageId = "Network Profile List";
+
+    fields
+    {
+        field(1; "CDN GUID"; Guid)
```

Comment:
"Cdn Guid", we typically don't uppercase the entire guid part. or maybe "Cdn Id" would be better